torch/csrc/jit/passes/bailout_graph.cpp

```diff
@@ -46,8 +51,15 @@ struct BailOutGraphBuilderForNode {
     return new_value;
   }
 
-  Value* getOrAddInputForValue(Value* v) {
+  Value* getOrAddInputForValue(Value* v, Value* default_val = nullptr) {
```

Comment:
This API is really confusing now.  If asked what this function does, I wouldn't be able to explain it easily.